cmd/troubleshoot/cli/analyze.go

```diff
@@ -59,8 +71,10 @@ func Analyze() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().String("url", "", "URL of the support bundle to analyze")
-	cmd.MarkFlagRequired("url")
+	cmd.Flags().String("bundle", "", "Filename of the support bundle to analyze")
```

Comment:
maybe "filename or URL of the support bundle to analyze"?

Assuming URLs still work

Reply:
Let's make sure to copy the capabilities of preflight and support-bundle here. Take a look at those two and how they set up the CLI.

My instinct is that preflight and support-bundle read their spec from arg[0]. not a flag, and we should copy that logic here for consistency.

Reply:
@marccampbell So preflight or support-bundle generation don't require an additional parameter to take a bundle input.  So can we nail down what we want here.  So it could be either:

`kubectl support-bundle analyze <url-for-analyzer-spec> --bundle=<url-or-file-bundle>`

or 

`kubectl support-bundle analyze <url-or-file-for-analyzer-spec> <url-or-file-bundle>`

At first glance it seemed confusing to offer the ability to download the support bundle through a URL, as our docs say to use `kubectl support-bundle <url-or-file-to-collector-spec>` to download a support-bundle to the local file system.  So it would seem logical the next step in the workflow is require a support-bundle on the local file system.  That being said, it should be one of the required parameters (so not a flag here?).   Although I can see people mixing up two positional parameters, which why a flag might be better.  This way all of our commands are consistent:

kubectl preflight <url>
kubectl support-bundle <url>
kubectl support-bundle analyze <url> --bundle=<file>

Note: kotsadm don't expose the URL to download the bundle so currently it seems there's no way for a user to get the URL.  

Thoughts?

Reply:
```kubectl support-bundle analyze <url-or-file-to-analyzer-spec> --bundle=<url-or-file-to-support-bundle>``` makes sense to me and feels consistent.

Reply:
ok, cool.  i'll go with that.